_posts/2025-02-28-this-week-in-rails.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Ignoring indexes in MySQL has never been so easy..."
+categories: news
+author: zzak
+og_image: assets/images/this-week-in-rails.png
+published: true
+date: 2025-02-28
+---
+
+
+Hey, [zzak](https://github.com/zzak) here. Last week was a short one, so we took the week off, let's catch up.
+
+Heads up: the **Rails World CFP** will open on **March 11th**.  
+The 2025 landing page is currently being built, but if you are going to submit a talk, keep an eye on: [rubyonrails.org/world/](https://rubyonrails.org/world)
+
+[Add Podman instructions for devcontainers](https://github.com/rails/rails/pull/54572)  
+This is not necessarily a change to Rails itself, but worth noting that you don't have to rely on Docker to use devcontainers and we actually use Podman to test the `rails new` flow in CI as well.
+
+[Add login_as(user) testing helper when generating authentication](https://github.com/rails/rails/pull/53708)  
+When using the Authentication generator, you now get `sign_in_as(user)` and `sign_out` test helper methods.
+
+[Introduce "ActiveSupport::ErrorReport#metadata_provider"](https://github.com/rails/rails/pull/54512)  
```

Comment:
```suggestion
[Introduce "ActiveSupport::ErrorReport#add_middleware"](https://github.com/rails/rails/pull/54512)  
```

Reply:
:joy: Thanks! I updated the PR but forgot to update the post